hazelcast/proxy/multi_map.py

```diff
@@ -445,6 +447,52 @@ def put(self, key: KeyType, value: ValueType) -> Future[bool]:
         request = multi_map_put_codec.encode_request(self.name, key_data, value_data, thread_id())
         return self._invoke_on_key(request, key_data, multi_map_put_codec.decode_response)
 
+    def put_all(self, map: typing.Dict[KeyType, typing.Sequence[ValueType]]) -> Future[bool]:
+        """Stores given values by associating with corresponding keys in dictionary format.
+
+        Warning:
+            This method uses ``__hash__`` and ``__eq__`` methods of binary form
+            of the key, not the actual implementations of ``__hash__`` and
+            ``__eq__`` defined in key's class.
+
+        Args:
+            map: the map corresponds to multimap entries.
+
+        Returns:
+            ``True`` if size of the multimap is increased, ``False`` if the
+            multimap already contains the key-value tuple.
```

Comment:
I tried to update. It might be wrong. [87a6da9](https://github.com/hazelcast/hazelcast-python-client/pull/573/commits/87a6da958a9cfc1bc57fff31af7995cf7081020e)

Reply:
This section is still in the PR, did you forget to delete it?